notification/src/handler/notification/notification.handler.js

```diff
@@ -6,6 +6,25 @@ const { getNotificationForTracking } = require('../../utils/commons')
 const ctp = require('../../utils/ctp')
 const mainLogger = require('../../utils/logger').getLogger()
 
+class CommercetoolsError extends Error {
+  constructor({ stack, message, statusCode }) {
+    super()
+    this.stack = stack
+    this.message = message
+
+    /*
+      We do not want to block notifications coming by Adyen.
+      So with this error it will be accepted by notification module by default.
```

Comment:
confusing statement. Is default always the same or actually differ depending on isRecoverable state?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/629/commits/6258cb50e64867b16529d185444426d37d649ac8